src/dataset-builder/DatasetBuilderUtils.test.ts

```diff
@@ -120,7 +120,6 @@ const criteriaGroup: CriteriaGroup = {
   criteria: anyCriteriaArray,
   mustMeet: true,
   meetAll: false,
-  count: 50,
```

Comment:
why do we not need counts on these types anymore?

Reply:
The count on the group is now computed dynamically whenever the group content changes. If we wanted to try and cache the group count then we could bring it back, but right now it doesn't have any usage. Caching could be helpful in the case where the user has multiple groups, edits one, and is returned back to the screen showing all groups. Right now it would need to recompute the counts for all groups even if they haven't been changed.